### PR TITLE
Include/exclude project properties

### LINKED ISSUE #58: Include/exclude project properties
I don't want to dump all the properties with **write-project-properties**.
Pls add includes/excludes configuration for this goal.

### PR DESCRIPTION
**Pull request by reda-alaoui**
Fix #58

### CODE REVIEW

**Comment by slachiewicz** [comment]
Please also add unit tests

**Comment by reda-alaoui** [comment]
@slachiewicz done :+1:

File: src/main/java/org/codehaus/mojo/properties/WriteProjectProperties.java [hunk 1]
```diff
@@ -40,6 +44,18 @@
...
 
+    /**
+     * Property keys to exclude.
+     */
+    @Parameter(property = "properties.excludedPropertyKeys")
+    private Set<String> excludedPropertyKeys;
+
+    /**
+     * Property keys to include.
+     */
```
Comment (slawekjaranowski) [inline src/main/java/org/codehaus/mojo/properties/WriteProjectProperties.java:55]: add `since`

**Review by slawekjaranowski — APPROVED** [review]

**Comment by slawekjaranowski** [comment]
@reda-alaoui - please rebase with current master - I fixed build by JDK 8 on macos

**Comment by reda-alaoui** [comment]
@slachiewicz done :)

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**